config/deploy/manifests.yaml

```diff
@@ -50,30 +57,20 @@ spec:
         description: AuthConfig is the schema for Authorino's AuthConfig API
         properties:
           apiVersion:
-            description: 'APIVersion defines the versioned schema of this representation
-              of an object. Servers should convert recognized schemas to the latest
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            description: 'APIVersion defines the versioned schema of this representation of an object. Servers should convert recognized schemas to the latest internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
```

Comment:
This seems to be down to the version of kustomize being used 3.5.4(https://github.com/Kuadrant/authorino/blob/main/Makefile#L73) vs 3.8.7 (https://github.com/Kuadrant/authorino-operator/blob/main/Makefile#L140)